### PR TITLE
JME DQM updates for Run3 (12_4_X)

### PR DESCRIPTION
**Pull request by esiam**
**#### PR description:**

Modify code for DQM, Validation and Harvesting steps for JME group.
  
Add kinematic variables, remove quantities that are not usefull, add efficiency plots and clean up files with trigger paths. An overview can be found  [here](https://indi.to/n3Z92). The changes were discussed with the JME trigger group ( @pallabidas , @sparedes) and also presented to the STEAM meeting ( @lbenato, @aknayak, @milosdjordjevic )   
                                                                                                                                                                                 
**#### PR validation:**

The changes were tested by running the DQM and Harvesting steps on RelVal samples. Besides they were tested by using the RunTheMatrix.py -l 11634.0. 

Please let me know, if more information is needed.

### CODE REVIEW

**Comment by emanueleusai** [comment]
+1

**Comment by qliphy** [comment]
@esiam please squash the commits into one

**Comment by esiam** [comment]
@qliphy thanks for your comment. Is this ok?

File: DQMOffline/Trigger/plugins/METMonitor.cc [hunk 1]
```diff
@@ -251,12 +330,30 @@ void METMonitor::analyze(edm::Event const& iEvent, edm::EventSetup const& iSetup
   }
   float deltaPhi_met_j1 = 10.0;
   float deltaPhi_j1_j2 = 10.0;
-
-  if (!jets.empty())
+  int nJets = jets.size();
```
Comment (perrotta) [inline DQMOffline/Trigger/plugins/METMonitor.cc:333]: same comment as for the similar structure in HTMonitor.cc

File: DQMOffline/Trigger/plugins/METMonitor.cc [hunk 2]
```diff
@@ -358,12 +473,31 @@ void METMonitor::analyze(edm::Event const& iEvent, edm::EventSetup const& iSetup
...
   deltaphij1j2ME_.numerator->Fill(deltaPhi_j1_j2);
+
+  nJetsME_.numerator->Fill(nJets);
+  if (!jets.empty()) {
+    for (int kk = 0; kk < nJets; kk++) {
+      jetPtME_.numerator->Fill(jets[kk].pt());
+      jetPhiME_.numerator->Fill(jets[kk].phi());
+      jetEtaME_.numerator->Fill(jets[kk].p4().eta());
+    }
+  }
```
Comment (perrotta) [inline DQMOffline/Trigger/plugins/METMonitor.cc:484]:
```suggestion
  for (auto const & jet : jets) {
    jetPtME_.numerator->Fill(jet.pt());
    jetPhiME_.numerator->Fill(jet.phi());
    jetEtaME_.numerator->Fill(jet.p4().eta());
  }
```

**Review by perrotta — COMMENTED** [review]
Could you also please provide a more meaningful title for this PR?

**Comment by esiam** [comment]
@perrotta thank you very much for your feedback. I am sorry for the late response.

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
@esiam I took the liberty to modify the title in order to have it a little more descriptive: please modify it further if you don't like it

**Comment by emanueleusai** [comment]
+1

**Comment by perrotta** [comment]
+1

**Pull request merged**